### PR TITLE
Add xsi format schemaLocation

### PR DESCRIPTION
**Pull request by angelozerr**
Add xsi format schemaLocation

See https://github.com/eclipse/lemminx/issues/825

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -231,6 +231,22 @@
           "markdownDescription": "Insert space before end of self closing tag. \nExample:\n  ```<tag/> -> <tag />```. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatspacebeforeemptyclosetag-) for more information.",
           "scope": "window"
         },
+        "xml.format.xsiSchemaLocationSplit": {
+          "type": "string",
+          "enum": [
+            "onNamespace",
+            "onLocation",
+            "none"
```
Comment (datho7561) [inline package.json:239]: Update to `"onElement"`, `"onPair"`, `"none"`;

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: fixed

File: package.json [hunk 2]
```diff
@@ -231,6 +231,22 @@
...
+        "xml.format.xsiSchemaLocationSplit": {
+          "type": "string",
+          "enum": [
+            "onElement",
+            "onPair",
+            "none"
+          ],
+          "markdownEnumDescriptions": [
+            "Split `xsi:schemaLocation` content after each namespace declaration",
+            "Split `xsi:schemaLocation` content after each location declaration",
```
Comment (datho7561) [inline package.json:243]: Maybe adjust these descriptions to be a bit more specific:
* "Put a new line after each namespace and URI, arranging them into one column"
* "Put a new line after each URI, arranging the content into pairs of namespace and URI"

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (angelozerr) [reply package.json]: fixed

**Review by datho7561 — APPROVED** [review]

**Pull request merged**